docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md

```diff
@@ -0,0 +1,851 @@
+---
+title: Promotions & Discounts feature integration
+description: Learn how to integrate the Promotions & Discounts feature into a Spryker project.
+template: feature-integration-guide-template
+---
+
+
+{% info_block errorBox %}
+
+The following feature integration guide expects the basic feature to be in place.
+
+The current sub-feature integration guide adds discount prioritization functionality..
+
+{% endinfo_block %}
+
+---
+
+## Install feature core
+
+Follow the steps below to install the Promotions & Discounts feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+
+| NAME           | VERSION            | INTEGRATION GUIDE                                                                                                                      |
+|----------------|--------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core   | {{page.version}}   | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/spryker-core-feature-integration.html)   |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker-feature/promotions-discounts {{page.version}} --update-with-dependencies
```

Comment:
we should use splitted modules at this command since this doc is different from regular IGs